go/darwin/Dockerfile.tmpl

```diff
@@ -38,17 +38,18 @@ COPY --from=build-llvm-apple /osxcross.tar.gz /tmp/osxcross.tar.gz
 RUN tar -xzf /tmp/osxcross.tar.gz -C / \
   && rm /tmp/osxcross.tar.gz
 {{- else }}
-ARG OSXCROSS_SDK_URL=https://storage.googleapis.com/obs-ci-cache/beats/MacOSX10.11.sdk.tar.xz
 ARG OSXCROSS_PATH=/usr/osxcross
 ARG OSXCROSS_REV=3034f7149716d815bc473d0a7b35d17e4cf175aa
 ARG SDK_VERSION=10.11
 ARG DARWIN_VERSION=15
 ARG OSX_VERSION_MIN=10.10
+
+COPY sdks/MacOSX10.11.sdk.tar.xz .    
 RUN \
```

Comment:
can't we `COPY sdks/MacOSX10.11.sdk.tar.xz tarballs/MacOSX${SDK_VERSION}.sdk.tar.xz`

here to avoid the `cp /app/MacOSX10.11.sdk.tar.xz tarballs/MacOSX${SDK_VERSION}.sdk.tar.xz` done below?

Reply:
Thanks for the feedback. Yes, I just needed to have it copy to `/tmp/osxcross/tarballs` and remove the mkdir from the RUN command below. This is simpler now. Next I'm going to open up a clean PR rather than this mess of trial & error commits. :)